topic0KeyPoint1/src/oop3_4/MyTime.java

```diff
@@ -79,17 +81,14 @@ public void setSecond(int second) throws IllegalArgumentException {
 	}
 
 	public static boolean isValidHour(int hour) {
-		if (hour >= 0 && hour <= 23)
-			return true;
-		else
-			return false;
+
+		return (hour >= 0 && hour <= 23) ? true : false;
```

Comment:
Thanks for the advice, I didn't realize that,  I suppose it is because I am not used to doing it that way. I'll be correcting it in a moment.